packages/native/src/components/Navigation/SlideIndicator/index.tsx

```diff
@@ -20,15 +21,20 @@ const Container = styled.View`
   position: relative;
 `;
 
-const Bullet = styled.TouchableOpacity`
+const bulletStyle = `
```

Comment:
Very minor but I think here rather than defining a normal template literal you can `import {css} from "styled-components/native` and use it like this:
```
const bulletStyle = css`
  /* ... your styles */
`;
```
One small advantage of this is that with the [proper configuration](https://styled-components.com/docs/tooling#syntax-highlighting), **syntax highlighting will work nicely**

Another advantage of this syntax — that doesn't apply in this case — is that you can style derive style from props with something like `${(p) => p.enabled ? "green" : "red}`
 
[More info here from the official styled-components docs](https://styled-components.com/docs/api#css)

Reply:
Useful thanks!